sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -197,7 +197,7 @@ using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
     RecognizedFormCollection receipts = await client.StartRecognizeReceiptsAsync(stream).WaitForCompletionAsync();
 
     // To see the list of the supported fields returned by service and its corresponding types, consult:
-    // https://aka.ms/azsdk/python/formrecognizer/receiptfields
+    // https://aka.ms/formrecognizer/receiptfields
```

Comment:
just curious, did the other languages switched to use this link too?

Reply:
Yep. Java and Python were using the python link, but agreed on switching as well. Not sure if their changes are already merged, though.